src/models.cairo

```diff
@@ -1,12 +1,13 @@
-use starknet::ContractAddress;
+use starknet::{ContractAddress};
+use core::option::Option;
```

Comment:
Option is not into the prelude? If not, let's try to update the edition to see if we can avoid to import that.